quamash/_windows.py

```diff
@@ -52,9 +52,6 @@ def _after_run_forever(self):
 		self.__event_poller.stop()
 
 
-baseclass = _ProactorEventLoop
```

Comment:
why? this will break stuff?

Reply:
Oh, this leaked from my custom branch ( see this issue https://github.com/harvimt/quamash/issues/45 )